Sources/HummingbirdXCT/HBXCTRouter.swift

```diff
@@ -50,10 +50,6 @@ struct HBXCTRouter<Responder: HBResponder>: HBXCTApplication where Responder.Con
             logger: self.logger,
             makeContext: self.makeContext
         )
-        // run the runBeforeServer processes before we start testing
-        for process in self.processesRunBeforeServerStart {
-            try await process()
-        }
         // if we have no services then just run test
         if self.services.count == 0 {
             return try await test(client)
```

Comment:
We should still run those processes here right?

Reply:
Yep you're right

Reply:
fixed